packages/_site/src/pages/components/routes_list_docs_components.ts

```diff
@@ -113,6 +113,12 @@ export const componentsList: I_ComponentsList[] = [
     link: "/components/hover_card/examples",
   },
 
+  {
+    title: "Popover",
+    description: "Popover Button. Click to display card's content to either the left, right, top or bottom of the button.",
```

Comment:
Change the description, to something like 

`Displays rich content in a portal, that is triggered by a customizable trigger component`

The popover isn't button specific.